app/controllers/users_controller.rb

```diff
@@ -311,7 +311,7 @@ def create
       return fail_with("login.reserved_username")
     end
 
-    if user = User.where(staged: true).find_by(email: params[:email].strip.downcase)
+    if user = User.where(staged: true).find_by_email(params[:email].strip.downcase)
```

Comment:
Usually, it is no recommended to make cosmetic changes like these as it pollutes the commit history. Also, `find_by_*` have been [deprecated in Rails 4](http://guides.rubyonrails.org/4_0_release_notes.html) and will be removed in Rails 5.